src/PactNet.Native/PactExtensions.cs

```diff
@@ -49,6 +49,25 @@ public static IPactBuilderV3 UsingNativeBackend(this IPactV3 pact, int? port = n
             return builder;
         }
 
+        /// <summary>
+        /// Establish a new pact using the native backend
+        /// </summary>
+        /// <param name="pact">Pact details</param>
+        /// <returns>Pact builder</returns>
+        /// <remarks>
+        /// If multiple mock servers are started at the same time, you must make sure you don't supply the same port twice.
+        /// It is advised that the port is not specified whenever possible to allow PactNet to allocate a port dynamically
+        /// and ensure there are no port clashes
+        /// </remarks>
+        public static IPactMessageBuilderV3 UsingNativeBackendForMessage(this IPactV3 pact)
```

Comment:
I wonder if it's worth defining an `IMessagePact` and `IMessagePactV3` etc so that the extension method is a bit neater. Then you could have:

```csharp
public static IPactMessageBuilderV3 UsingNativeBackend(this IMessagePactV3 pact)
```

which would means there wouldn't be an API difference between message and non-message pacts.

Reply:
Let me see...
We would them call it like that :
IMessagePactV3 pactMessageV3 = Pact.V3Message("Event API Consumer V3 Message", "Event API V3 Message", _config);
_pactMessage = pactMessageV3.UsingNativeBackendForMessage();

And the Pact class would then implement the new IMessagePactV3 ?

In that case, not sure we need to have an IMessagePact interface. All interaction model should implement the IPact interface, no?

Reply:
I tried to add a IMessagePact and IPactMessageV3 with a UsingNativeBackend()... 
More duplicated code but let's see what we prefer.